app/views/bands/new.html.erb

```diff
@@ -0,0 +1,14 @@
+
+<%= form_for (@band) do |f| %>
+  <div class="field">
+    <%= f.label :name, "Band Name" %>
+    <%= f.text_field :name %>
+  </div>
+  
+  <div class="field">
+    <%= f.label :biography %>
+    <%= f.text_area :biography %>
+  </div>
+
+  <%= f.submit "Add Local Band"%>
+<%end%>
```

Comment:
this would appear as an empty form with no text on the page. Minimally add some H1 to inform the user what this is for